scripts/publish-cmds.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+if [[ -z "$1" ]]; then
+  echo -e "version is not set, run:\n"
+  echo "$0 [version] [prod|dev]"
```

Comment:
This comment confused me
```suggestion
  echo "$0 [version] [v0.x.x]"
```